### ISSUE TITLE
Stepping up as core dev

### ISSUE THREAD

**Issue by Zughy** [LibreSprite/LibreSprite#148]
Yes, I stated multiple times that I'm nowhere near knowing C++ as some professional dev does, but after thinking about it for a while I think I can learn and survive. These are the reasons that lead me to this request:
- Maujin is doing basically all by himself, and it seems to me he has little to no time because of work. I don't want to stress him, but I also don't want to see this project going stale when there is someone with some free time (me)
- Talking about work, I do use LibreSprite to work (alongside with Aseprite, I have one on a PC and one on another). This means I'm pretty motivated to improve it
- I contributed to a C++ project before (namely Minetest, [here](https://github.com/minetest/minetest/commits?author=Zughy)) and I have a pretty solid knowledge about coding in general thanks to some old personal projects, even if some languages are reeeeally rusty. I've used Ruby, Java, JS, Python, C#, Lua and GDscript. In short, I know where to put my hands
- I like to code, but I'm being the 3D modeller in my current project (we're a team). Working on LibreSprite allows me to do something I like
- Even more important, I like to learn...
- ...and to keep things organised. All those untagged issues, just waiting for some labels :eye:
- I also fully support free software philosophy and I'm doing my best to promote LibreSprite. Being able to impact the development would be another way to support the cause
- Worst case scenario, when I'm not sure about something I have a few professional developer friends who can help me, so I shouldn't make explode anything
- And worst worst case scenario, I'm also pretty good with designs, so I can do mock-ups and/or manage any UI change

In short: I care about this project, I have time and I'm pretty decent at coding

Tagging @maujin111 @PerryHugh and @MintStudios to know what they think about it

**Comment by PerryHugh** [comment]
I can't dedicate any time to the project, so I should honestly be dropped. At this point I was just hoping someone would go all out on this project. Go for it man.

**Comment by maujin111** [comment]
I also agree that you be promoted to maintainer

**Comment by Zughy** [comment]
Closing since... well, I am now :D
